.vscode/settings.json

```diff
@@ -17,6 +17,6 @@
     // "editor.formatOnSave": true
   },
   "extensions.ignoreRecommendations": true,
-  "files.autoSave": "off",
+  "files.autoSave": "afterDelay",
```

Comment:
Was there a reason you set this for the project? I think perhaps this should be a user setting, because we don't want to adjust everyone else's settings unless there's a specific reason to.

Reply:
That was my mistake. I should not upload this file.